renovate.json

```diff
@@ -1,7 +1,7 @@
 {
 	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
 	"extends": [
-		"config:base",
+		"config:recommended",
```

Comment:
base was renamed to recommend: https://github.com/renovatebot/renovate/issues/23326